jalib/jfilesystem.cpp

```diff
@@ -183,8 +186,11 @@ jalib::string jalib::Filesystem::GetProgramName()
 
 jalib::string jalib::Filesystem::GetProgramPath()
 {
-  static jalib::string value = _GetProgramExe();
-  return value;
+  static jalib::string *value = NULL;
+  if (value == NULL) {
```

Comment:
Suggest adding comment:
// Technically, this is a memory leak, but value is tatic and so it happens only once.